src/js/mixins/evented.js

```diff
@@ -444,8 +445,14 @@ const EventedMixin = {
     const type = event && typeof event !== 'string' ? event.type : event;
 
     if (!isValidEventType(type)) {
-      throw new Error(`Invalid event type for ${objName(this)}#trigger; ` +
-        'must be a non-empty string or object with a type key that has a non-empty value.');
+      const error = `Invalid event type for ${objName(this)}#trigger; ` +
+        'must be a non-empty string or object with a type key that has a non-empty value.';
+
+      if (event) {
+        (this.log || log).error(error);
```

Comment:
should we return no matter what if we don't get a valid event type?

Reply:
I think to reduce potential breakage, we want to try and continue through to the next part here and not return early.
Though, if they got here there's probably something wrong, but it could still be working in some way, for better and worse.